builder/docker/javascript/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM suborbital/subo:dev as subo
+
+FROM node:16-bullseye-slim
+
+WORKDIR /root
+RUN mkdir runnable; mkdir suborbital
+
+ENV PATH="/root/.cargo/bin:$PATH"
+
+RUN apt-get update && \
+    apt-get install pkg-config git build-essential libssl-dev clang cmake curl -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
+    rustup target install wasm32-wasi && \
+    cargo install wasmtime-cli cargo-wasi && \
```

Comment:
What is wasmtime-cli needed for?

Reply:
It's needed to build Javy (at least according to the README).